docs/dev-tools/chopsticks/index.md

```diff
@@ -0,0 +1,437 @@
+---
+title: Using chopsticks for Substrate based chains.
+description: Configure and fork Substrate-based blockchains locally with Chopsticks. Learn installation, configuration, and usage for streamlined blockchain development.
+---
+
+# Chopsticks
```

Comment:
This file should not be the `index.md` file. If you want to only have one page on chopsticks, then you don't need a directory for it, instead under `dev-tools`, you can create a file named `chopsticks.md`:

```
# docs/dev-tools/.pages
title: Developer Tools
nav:
- index.md
- 'Interacting with Chopsticks': 'chopsticks.md' # Adding the title here dictates what we see in the left navigation bar. Then, the title at the top of the `chopsticks.md`, that is for SEO purposes, so it can be longer if needed
```


If you're planning on having multiple pages for chopsticks, then I'd say keep the `chopsticks` directory and follow a structure like:

```
# docs/dev-tools/chopsticks/.pages`
title: Chopsticks
nav:
- index.md
- 'Chopsticks Overview': 'overview.md'
- 'Use Chopsticks for XCM Testing': 'xcm-testing.md'
```

Reply:
Hello @0xLucca! Based on the conversation above, can you please confirm the desired directory structure? I can help with updating the `.pages`, etc., but I need to know what you're trying to achieve before I can update anything. Thanks!

Reply:
Hey @dawnkelly09! For now, I'll stick to something similar to the following:
```
# docs/dev-tools/chopsticks/.pages`
title: Chopsticks
nav:
- index.md
- 'Overview': 'overview.md'
- 'OpenGov Tutorial': 'chopsticks-open-gov.md'
```

In the future if we have more tutorial we can move it to a tutorials subsection. What do you think?

Reply:
This PR only updates `overview.md` for Chopsticks. I've got a separate branch for `chopsticks-open-gov.md` so we can keep adding content to the live page while keeping changes modular and easier to review.